### PR TITLE
Update telemetry plugins

### PR DESCRIPTION
**Pull request by rroux-gpsw**

### CODE REVIEW

File: Speedometer/src/Speedometer.js [hunk 1]
```diff
@@ -209,57 +229,75 @@ ForgePlugins.Speedometer.prototype = {
...
 
             // draw end of moving gauge
-            ctx.fillStyle = this.plugin.options.gauge.color;
-            ctx.beginPath();
-            ctx.arc(center + radius * (1 - ((this.plugin.options.gauge.tracker / 2 + 2) / radius)) * Math.cos(angle), center + radius * (1 - ((this.plugin.options.gauge.tracker / 2 + 2) / radius)) * Math.sin(angle), (this.plugin.options.gauge.tracker / 2) + 4, 0, 2 * Math.PI);
-            ctx.fill();
-            ctx.lineWidth = 2;
-            ctx.stroke();
-            ctx.closePath();
+            // ctx.fillStyle = this.plugin.options.gauge.color;
```
Comment (aberthet-gpsw) [inline Speedometer/src/Speedometer.js]: To remove

**Review by aberthet-gpsw — COMMENTED** [review]
Good improvements, both visually and in the code. Need some cleaning in commented code and it will be good to go.

File: Speedometer/src/Speedometer.js (hunk 1 above)
Comment (rroux-gpsw) [reply Speedometer/src/Speedometer.js]: Yes! I'll do it

**Review by aberthet-gpsw — APPROVED** [review]
LGTM

**Pull request merged**